.prettierignore

```diff
@@ -0,0 +1,3 @@
+framer/Material-UI.framerfx/design/*
```

Comment:
Should the files in `.prettierignore`  be in the `.gitignore` too and moved to the .eslintignore?

Reply:
No, they're written by Framer, so prettying them every time is pointless, but they still need to be included in the package AFAIK.

Reply:
So we do need to keep them in the git repository? They can't be generated before publishing?

Reply:
Sorry, deleted my comment so I could  expand on it after lunch. Didn't realise you'd already replied.

They're written by Framer, so prettying them every time is pointless, but some of them still need to be included in the package:

 - `build` is already in `.gitignore`.
 - `design` contains the project data. It isn't needed in order to publish the components, but it's useful for testing the built components in Framer.
 - `canvas.tsx` I just tried deleting, and Framer neither complained about it being missing, nor recreated it. It might be left over from before converting this to a folder-backed project? Removed from `.prettierignore`.

I've moved `.prettierignore` into the framer directory now that the yarn script is there.

Reply:
So we can move them into .eslintignore? .prettierignore shouldn't have any effect to being with: https://github.com/mui-org/material-ui/blob/3f1bf78d2f674ce89f6a2bfeae32ac2add93a9a2/scripts/prettier.js#L23

Reply:
It isn't using the script

Reply:
Yep saw that. This is fine for me then for now. We can always improve later (I guess we want some form of CI for this anyway).